README.md

```diff
@@ -117,3 +120,63 @@ The test data, because it is big and not code, is separated from the main `data_
         git submodule update
     
 The sub-repository will be cloned in the directory `test_resources`.
+
+## Caching with Redis
+
+   There is a deployment.cfg file in the data_api source directory that contains several target deployment locations.
+   By default, services and tests will use dir_nocache, which uses the local file test data and will not require or use caching.
+   If you would like to test with caching enabled, you can set the KB_DEPLOY_URL environment variable to one of the targets
+   in the deployment.cfg file, for instance dir_cache.  This sets the redis_host to localhost and the redis_port to the default port.
+   You can edit this file to change settings before starting Redis and the Data API services.
+   
+   There is also a redis.conf file located in the data_api source directory for running with Redis locally.
+   
+   Installation of Redis:
+   
+        apt-get install redis
```

Comment:
I was going to say to just let Redis handle this, but their instructions are for a source build!
So let's just add:
* Linux:
    - Debian/Ubuntu: `apt-get install redis`
    - RedHat/CentOS: `yum install redis`
* OSX:
  - Homebrew: `brew install redis`
  - Macports: `port install redis`

For other operating systems, see the [Redis homepage](https://redis.io)